unicorn_binance_websocket_api/unicorn_binance_websocket_api_manager.py

```diff
@@ -154,7 +154,7 @@ def __init__(self,
                  enable_stream_signal_buffer=False):
         threading.Thread.__init__(self)
         self.name = "unicorn-binance-websocket-api"
-        self.version = "1.28.0.dev"
+        self.version = "1.28.1.dev"
```

Comment:
*.dev is the development stage. Only when we create a new package we update this and remove the "dev".  You dont need to care about this! Its done during the build of the package!

Reply:
Can you undo the version changes? Then I would merge this.